### PR TITLE
Upgrade locutus

### PR DESCRIPTION
**Pull request by clyang82**
fixed: #58 
fixed: https://github.com/brancz/locutus/issues/44

Signed-off-by: clyang82 <chuyang@redhat.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # Build the manager binary
-FROM golang:1.13.3-alpine3.10 as builder
-# TODO(kakkoyun): Upgrade Go version.
+FROM golang:1.14-alpine as builder
```
Comment (kakkoyun) [inline Dockerfile:2]: Why not upgrade to 1.15?

**Review by kakkoyun — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (clyang82) [reply Dockerfile:2]: go.mod is using 1.15 so just keep to use 1.14. we should upgrade to use 1.15. but it seems we need to upgrade to locutus firstly.

Comment (kakkoyun) [reply Dockerfile:2]: I guess you wanted to say go.mod file is using 1.14 :) Go mod file version is the minimum supported version. If one doesn't use any new features, that version doesn't need to be upgraded. So it's safe to upgrade container version without touching the go.mod file. I think we can just move to the latest stable even, 1.16

Comment (kakkoyun) [reply Dockerfile:2]: We can address this in another PR.

**Pull request merged**